sdk/template/ci.yml

```diff
@@ -23,6 +22,8 @@ pr:
   paths:
     include:
     - sdk/template/
+    exclude:
```

Comment:
Do we need to explicitly exclude this path?

Reply:
Am not sure its necessary, I just wanted to be doubly sure.

Reply:
I'd like to not have it be added explicitly unless we know there is a reason.